test/metabase/test/data.clj

```diff
@@ -301,3 +303,9 @@
 
 (defmacro with-venue-category-fk-remapping [remapping-name & body]
   `(do-with-venue-category-fk-remapping ~remapping-name (fn [] ~@body)))
+
+(defn get-field-definitions
+  "Gets the field definitions for the given table-id"
+  [table-id]
+  (map (partial into {})
+       (db/select [Field :name :database_type :base_type] :table_id table-id)))
```

Comment:
I don't really think this one-off function really reaches the threshold of belonging in a common test util namespace -- we already have several thousand lines of common test util functions and if we added every random helper function we'd end up with tens of thousands. I'd just put this in the namespace next to the test that uses it.

Reply:
OK, removed it.